src/test/java/tests/workers/imports/TestNG/ClassNameTestNGHandlerTest.java

```diff
@@ -11,11 +11,11 @@
 
 public class ClassNameTestNGHandlerTest implements IHandlerTest {
 
-    private JavaJUnitTestNG javaJUnitTestNGHandler;
+    private MavenSurefireHandler mavenSurefireHandlerHandler;
 
     @Override
     public Handler getHandler() {
-        return javaJUnitTestNGHandler;
+        return mavenSurefireHandlerHandler;
     }
 
     @Override
```

Comment:
I think you need to refactor resources folder, because you have files for JUnit and TestNG folders right now